lib/travis/services/find_admin.rb

```diff
@@ -10,11 +10,11 @@ class FindAdmin < Base
       include Travis::Logging
 
       register :find_admin
+      NUM_CANDIDATES = 15
```

Comment:
Awesome work @aakritigupta! Thank you for fixing this issue!

I have a question regarding this number: is there a rationale behind precisely choosing 15 here?

Thanks again!